src/provider/provider_os_memory.c

```diff
@@ -341,6 +341,22 @@ validatePartitions(umf_os_memory_provider_params_t *params) {
     return UMF_RESULT_SUCCESS;
 }
 
+static umf_result_t os_get_min_page_size(void *provider, void *ptr,
```

Comment:
don't we already have a function like this?

Reply:
Yes, I have to declare it before use. Removed the redundant declaration below, though.